requirements/base.txt

```diff
@@ -28,7 +28,7 @@ django==3.2.15
     #   django-filter
     #   django-model-utils
     #   djangorestframework
-django-bootstrap3==21.2
+django-bootstrap3==22.1
```

Comment:
Dropped support for `Django 2.2`. https://github.com/zostera/django-bootstrap3/blob/main/CHANGELOG.md#221-2022-08-08